packages/angular-17/projects/components/README.md

```diff
@@ -0,0 +1,76 @@
+[![Storybook](https://img.shields.io/badge/docs-storybook-ff69b4)](https://tds-storybook.tegel.scania.com/)
+![](https://img.shields.io/github/license/scania-digital-design-system/tegel)
+![Status: Beta](https://img.shields.io/badge/status-beta-red)
+![npm](https://img.shields.io/npm/v/%40scania%2Ftegel-angular-17)
+
+# @scania/tegel-angular-17
+This is a ongoing project with the purpose to simplify the integration of Tegel in Angular apps.
+
+This guide [here](https://stenciljs.com/docs/angular) has served as base for the configuration.
+
+Official website: https://tegel.scania.com/
+
+Storybook: https://tds-storybook.tegel.scania.com/
+
+The design system supports the design and development of digital solutions at Scania. The purpose is to secure a coherent, premium brand and user experience across all of Scania's digital touchpoints.
+
+## Installation
+
+
+1. Run `npm install @scania/tegel-angular`
+2. In your root module (app.component.ts) import the `TegelModule`:
+
+```ts
+import { Component } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import { TegelModule } from '@scania/tegel-angular';
```

Comment:
Same here, should it reflect the tegel-angular-17 package?

Reply:
Good finding!